src/web/tools/VelocityTracker.ts

```diff
@@ -84,7 +84,7 @@ export default class VelocityTracker {
     return null;
   }
 
-  public getVelocity(): [number, number] {
+  public get velocity() {
```

Comment:
Doesn't this change the return type?

Reply:
It does to `number[]`. Changed in [b7b4eaa](https://github.com/software-mansion/react-native-gesture-handler/pull/3303/commits/b7b4eaab58d6732b0fd653b24403af43ffd8ca14)